.github/workflows/ci.yml

```diff
@@ -24,17 +24,41 @@ jobs:
         working-directory: docs
         run: npm install
 
-      - name: Lint Files
-        run: node Makefile lint
+      - name: Lint Files (eslint)
+        uses: trunk-io/trunk-action@v1
+        with:
+
+          # Run on everything except the docs folder.
+          arguments: --ignore=docs/** --filter=eslint
+          check-mode: all
+
+      - name: Lint Files (other)
+        uses: trunk-io/trunk-action@v1
+        with:
+
+          # Run on everything except the docs folder.
+          arguments: --ignore=docs/** --filter=-eslint
```

Comment:
The steps " Lint Files (eslint)" and "Lint Files (other)" have the same arguments and only differ for the presence of the `check-mode: all` setting. Did you mean to include `--filter=eslint` only in the first step to run ESLint on all files, and have the other linters run only on changed files in the other step?